mail_composer_cc_bcc/models/mail_thread.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2023 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, tools
+
+
+def format_emails(partners):
+    emails = [
+        tools.formataddr((p.name or "False", p.email or "False")) for p in partners
+    ]
+    return ", ".join(emails)
```

Comment:
weird, can you check when did we introduce that duplicate?

should be `from .mail_mail import format_emails`

Reply:
Probably I duplicated it when migrating. I just changed to `from .mail_mail import format_emails`